src/assets/javascript/cookies.js

```diff
@@ -75,7 +77,50 @@ window.govSigin = window.govSigin ?? {};
   }
 
   function isOnCookiesPage() {
```

Comment:
further to the comment above a way to spilt it could be as follows:

```
function cookiesPageInit() {
    var cookie = getCookie(COOKIES_PREFERENCES_SET);
    var analyticsValue = false;

    if (!cookie) {
      setCookie(COOKIES_PREFERENCES_SET, { analytics: false });
    } else {
      analyticsValue = JSON.parse(cookie).analytics;
    }

    document.querySelector("#policy-cookies-accepted").checked = analyticsValue;
    document.querySelector("#policy-cookies-rejected").checked =
      !analyticsValue;

    document.querySelector("#save-cookie-settings").addEventListener(
      "click",
      function (event) {
        event.preventDefault();
        var selectedPreference = document.querySelector(
          '#radio-cookie-preferences input[type="radio"]:checked'
        ).value;
        setCookie(COOKIES_PREFERENCES_SET, {
          analytics: selectedPreference === "true",
        });
        showElement(document.querySelector("#save-success-banner"));
        window.scrollTo(0, 0);
      }.bind(this)
    );

    document.querySelector("#go-back-link").addEventListener(
      "click",
      function (event) {
        event.preventDefault();
        window.history.back();
      }.bind(this)
    );
  }

  function isOnCookiesPage() {
    return window.location.pathname !== "/cookies";
  }
```